app/uk/gov/hmrc/lisaapi/models/package.scala

```diff
@@ -93,7 +95,10 @@ package object models {
       "^(Purchase failed|Purchase completed)$".r,
       "error.formatting.propertyPurchaseResult"
     )
-    
+    val isaManagerName: Reads[LisaManagerName] = Reads.pattern("^[a-zA-Z0-9 '/,&().-]{1,50}$".r, "error.formatting.lisaManagerName")
```

Comment:
Should this have been `lisaManagerName` too, or do you think it should be `isaManagerName` by this point? If it's the reads I lean to the former.